SPECS/iptables/iptables

```diff
@@ -60,6 +61,8 @@ iptables -t nat -F
 iptables -t nat -X
 iptables -t mangle -F
 iptables -t mangle -X
+iptables -t raw -F
```

Comment:
For this PR, please remove the raw table flush and rule removal. These changes may be a bit unsteady for a stable release.